src/odin/narrativebuilder.cc

```diff
@@ -4531,6 +4581,52 @@ NarrativeBuilder::FormRelativeThreeDirection(DirectionsLeg_Maneuver_Type type,
   }
 }
 
+std::string NarrativeBuilder::FormLandmarkType(RouteLandmark_Type type,
```

Comment:
instead of this function, we can cast the `RouteLandmark_Type` to `uint8_t` and index into the phrases directly